plutus-ledger/src/Ledger/Tx/Internal.hs

```diff
@@ -228,48 +229,52 @@ instance OpenApi.ToSchema TxOut where
 
 instance Pretty TxOut where
   pretty (TxOut (C.TxOut addr v d rs)) =
-    hang 2 $ vsep
+    hang 2 $ vsep $
       ["-" <+> pretty (fromCardanoTxOutValue v) <+> "addressed to"
       , pretty (fromCardanoAddressInEra addr)
-      , "with" <+> case fromCardanoTxOutDatum d of
-          PV2.NoOutputDatum      -> "no datum"
-          PV2.OutputDatum dv     -> "inline datum" <+> viaShow dv
-          PV2.OutputDatumHash dh -> "datum hash" <+> pretty dh
-      , "and with" <+> case rs of
-          C.ReferenceScript _ (C.ScriptInAnyLang _ s) ->
-            "reference script hash" <+> viaShow (C.hashScript s)
-          C.ReferenceScriptNone -> "no reference script"
       ]
+      <> case fromCardanoTxOutDatum d of
+          PV2.NoOutputDatum      -> []
+          PV2.OutputDatum dv     -> ["with inline datum" <+> viaShow dv]
+          PV2.OutputDatumHash dh -> ["with datum hash" <+> pretty dh]
+      <> case rs of
+          C.ReferenceScript _ (C.ScriptInAnyLang _ s) ->
+            ["with reference script hash" <+> viaShow (C.hashScript s)]
+          C.ReferenceScriptNone -> []
 
 -- | A Babbage-era transaction, including witnesses for its inputs.
 data Tx = Tx {
-    txInputs          :: [TxInput],
+    txInputs           :: [TxInput],
     -- ^ The inputs to this transaction.
-    txReferenceInputs :: [TxInput],
+    txReferenceInputs  :: [TxInput],
     -- ^ The reference inputs to this transaction.
-    txCollateral      :: [TxInput],
+    txCollateralInputs :: [TxInput],
     -- ^ The collateral inputs to cover the fees in case validation of the transaction fails.
-    txOutputs         :: [TxOut],
+    txOutputs          :: [TxOut],
     -- ^ The outputs of this transaction, ordered so they can be referenced by index.
-    txMint            :: !Value,
+    txReturnCollateral :: Maybe TxOut,
+    -- ^
```

Comment:
Missing comment here and below

Reply:
D'oh I was sure adding empty comments would let me not to forget them later!